app/src/main/java/com/nguyenxuansangvn/demomvvm/data/repository/RepositoriesRepository.kt

```diff
@@ -0,0 +1,16 @@
+package com.nguyenxuansangvn.demomvvm.data.repository
+
+import com.nguyenxuansangvn.demomvvm.data.ApiService
+import com.nguyenxuansangvn.demomvvm.data.model.Repositories
+import com.nguyenxuansangvn.demomvvm.data.source.remote.UserRemoteDataSource
+import io.reactivex.rxjava3.core.Observable
+
+class RepositoriesRepository(
```

Comment:
```suggestion
class GithubRepoRepository (
```
thêm 1 cái interface GithubRepoRepositoryType để class này implement lại